sentry/src/main/java/io/sentry/ScopeType.java

```diff
@@ -3,5 +3,8 @@
 public enum ScopeType {
   CURRENT,
   ISOLATION,
-  GLOBAL;
+  GLOBAL,
+
+  // TODO do we need a combined as well so configureScope
+  COMBINED;
```

Comment:
This would allow users to use `.configureScope()` and be able to access the combined view of all scopes relevant, so they can read the same way `SentryClient` would.

Reply:
Yeah, cross platform SDKs (via `InternalSentrySdk`) are the primary use case atm. I'm about to play around with this but I'm gonna need help testing cross platform SDKs after the change.